x-pack/metricbeat/module/iis/application_pool/_meta/fields.yml

```
@@ -1,10 +1,76 @@
 - name: application_pool
   type: group
-  release: beta
+  release: ga
   description: >
     application_pool
   fields:
     - name: name
       type: keyword
       description: >
         application pool name
+    - name: process
+      type: group
+      description: >
+        process details
+      fields:
+        - name: pid
+          type: long
+          description: >
+             process id
+        - name: handle_count
+          type: long
+          description: >
+            handle count
```

These descriptions are intended for documentation, please use full sentences, e.g:
```suggestion
            Number of handles in use by the server.
```